rdrf/registry/patients/models.py

```diff
@@ -84,11 +84,15 @@ def __unicode__(self):
 class PatientManager(models.Manager):
 
     def get_by_registry(self, registry):
-        return self.model.objects.filter(rdrf_registry__in=registry)
+        registries = registry if hasattr(registry, '__iter__') else (registry,)
```

Comment:
Yes, didn't want to break/find out intent of previous code so kept it backwards compatible.
Changed is as suggested.